adb_graphics/default_specs.yml

```diff
@@ -625,27 +625,29 @@ hlcy: # Helicity
     title: 2-5km Updraft Helicity
   mn02: &hlcy_mn02 # Hourly minimum of updraft helicity over 0-2 km layer
     <<: *hlcy
-    clevs: !!python/object/apply:numpy.arange [12.5, 150.5, 12.5]
+    clevs: [-300,-275,-250,-225,-200,-175,-150,-125,-100,-75,-50,-25]
```

Comment:
Can we use a range here? And maybe an anchor so that the ones below can just reference this setting and all show up the same? Just to reduce duplication.

Reply:
These fields are already "double anchored".  I might need some advice on how to get a third level of anchoring in, or to modify the existing anchors.